__tests__/main.test.ts

```diff
@@ -280,6 +280,20 @@ describe("VIB", () => {
       expect(pipeline).not.toEqual("")
     })
 
+    it("Create not default pipeline with parameter with a file", async () => {
```

Comment:
Better `Create not default pipeline with runtime parameters from file`

Reply:
Update